source/ui/Image.js

```diff
@@ -174,12 +186,17 @@
 		srcChanged: function () {
 			var src = enyo.ri.selectSrc(this.src);
 			if (this.sizing) {
-				this.applyStyle('background-image', src ? 'url(' + enyo.path.rewrite(src) + ')' : 'none');
+				var placeholder = this.placeholder ? 'url(\'' + enyo.path.rewrite(this.placeholder) + '\')' : '';
+				var url= (src ? ( placeholder ? 'url(\'' + enyo.path.rewrite(src) + '\'),' : 'url(\'' + enyo.path.rewrite(src) + '\')') : '');
+				this.applyStyle('background-image', url+placeholder);
 			} else {
 				if (!src) {
 					// allow us to clear the src property
 					this.setAttribute('src', '');
 				} else {
+					if (this.placeholder) {
+						this.applyStyle('background-image', 'url(\'' + enyo.path.rewrite(this.placeholder) + '\')');
+					}
```

Comment:
This block is duplicating `placeholderChanged'. While we could replace it with a call to that method, I think it'd be better to add a call in `create` before `srcChanged` which would give the same effect. More importantly, by adding the call in `create`, we also allow for setting `placeholder` and not `src`. With the current code, nothing would be displayed (if both were set at create-time) because this block would not be hit.